src/PhpOfByBundle/Controller/DefaultController.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace PhpOfByBundle\Controller;
+namespace PhpOfBy\PhpOfByBundle\Controller;
```

Comment:
Why do you change namespaces here? May be it will be better to move bundle to sub folder before changing namespace? I think we will have more than one bundle in our application. Also it conflicts with Symfony best practices.